src/terminfo/parm.rs

```diff
@@ -100,10 +91,8 @@ pub fn expand(cap: &[u8], params: &[Param], vars: &mut Variables)
     let mut stack: Vec<Param> = Vec::new();
 
     // Copy parameters into a local vector for mutability
-    let mut mparams = [
-        Number(0), Number(0), Number(0), Number(0), Number(0),
-        Number(0), Number(0), Number(0), Number(0),
-    ];
+    let mut mparams = [Number(0), Number(0), Number(0), Number(0), Number(0), Number(0),
```

Comment:
I *really* don't like this alignment. Is there any way to put the `Number`s on a new line?